backend/src/routes/api/featurestores/featureStores.ts

```diff
@@ -0,0 +1,273 @@
+import { FastifyReply } from 'fastify';
+import * as https from 'https';
+import { KubeFastifyInstance, OauthFastifyRequest } from '../../../types';
+import { getAccessToken, getDirectCallOptions } from '../../../utils/directCallUtils';
+import { createCustomError } from '../../../utils/requestUtils';
+import { getNamespaces } from '../../../utils/notebookUtils';
+import {
+  parseNamespacesData,
+  findRegistryUrlForFeatureStore,
+  extractServiceInfo,
+  constructRegistryProxyUrl,
+  createFeatureStoreResponse,
+  filterEnabledCRDs,
+  getClientConfigsBatched,
+  fetchFromRegistry,
+  fetchConfigMap,
+  handleError,
+  makeAuthenticatedHttpRequest,
+  type FeatureStoreCRD,
+} from './featureStoreUtils';
+
+interface FeatureStoreResponse {
+  featureStores: FeatureStore[];
+}
+
+interface FeatureStore {
+  name: string;
+  project: string;
+  registry: {
+    path: string;
+  };
+  namespace?: string;
+  status: {
+    conditions: Array<{
+      type: string;
+      status: string;
+      lastTransitionTime: string;
+    }>;
+  };
+}
+
+interface CustomObjectResponse {
+  body: {
+    items?: FeatureStoreCRD[];
+  };
+}
+
+function buildFeatureStoreResponse(featureStores: FeatureStore[]): FeatureStoreResponse {
+  return { featureStores };
+}
+
+async function fetchFeatureStoreCRDs(
```

Comment:
it should be CR instead of CRD
```suggestion
async function fetchFeatureStoreCRs(
```